client.go

```diff
@@ -491,7 +481,7 @@ func (c *Client) sendHub() {
 				// And get rid of the timeout workaround.
 				c.LocalKite.Log.Error("error sending %q: %s", msg, err)
 
-				if err == sockjsclient.ErrSessionClosed {
+				if err == sockjsclient.ErrSessionClosed || strings.Contains(err.Error(), errClosing) {
```

Comment:
instead of comparing strings we can listen on closeChan and return

```
if err != nil {
    select {
    case <-s.closeChan:
        return 
    default:
        if err == sockjsclient.ErrSessionClosed {
            return
        }
    }
}
```

wdyt?